src/node/desktop/test/unit/main/window-tracker.test.ts

```diff
@@ -64,17 +64,17 @@ describe('window-tracker', () => {
   });
   it('duplicate name replaces the original', () => {
     const tracker = new WindowTracker();
-    const oneWin = new DesktopBrowserWindow(false, false, 'some name');
+    const oneWin = new DesktopBrowserWindow(false, false, false, 'some name');
     tracker.addWindow('one', oneWin);
-    const twoWin = new TestGwtWindow(false, false, 'the gwt window');
+    const twoWin = new TestGwtWindow(false, false, false, 'the gwt window');
     tracker.addWindow('one', twoWin);
     assert.equal(tracker.length(), 1);
     const result = tracker.getWindow('one');
     assert.deepEqual(result, twoWin);
   });
   it('delete window removes it from map', async function () {
     const tracker = new WindowTracker();
-    const oneWin = new DesktopBrowserWindow(false, false, 'some name');
+    const oneWin = new DesktopBrowserWindow(false, false, false, 'some name');
```

Comment:
Not for this PR, but it might be nice to define an interface for creating these objects with named parameters, so one could write:

```
new DesktopBrowserWindow({
  param1: false,
  param2: false,
});
```

Right now, it's hard to tell at a glance what the parameters actually mean or do in their usages.

Reply:
(discussed a bit in https://www.typescriptlang.org/docs/handbook/2/functions.html#parameter-destructuring)

Reply:
Agree, in C++ I usually comment calls like that as a cheap workaround: `(false/*foo*/, true/*bar*/) but better to do like you said.